corehq/apps/enterprise/tests/test_apis.py

```diff
@@ -0,0 +1,111 @@
+import json
+from django.test import TestCase, RequestFactory
+from django_prbac.models import Role, Grant, UserRole
+from corehq import privileges
+import base64
+from datetime import datetime, timezone
+from corehq.apps.users.models import WebUser, CommCareUser, HQApiKey
+from corehq.apps.accounting.tests import generator
+from corehq.apps.domain.models import Domain
+from corehq.apps.es.tests.utils import es_test
+from corehq.apps.es.forms import form_adapter
+from corehq.apps.enterprise.api.resources import FormSubmissionResource
+from corehq.form_processor.tests.utils import create_form_for_test
+
+
+@es_test(requires=[form_adapter])
+class FormSubmissionResourceTests(TestCase):
+    def test_happy_path(self):
```

Comment:
maybe rename to `test_resource_is_accessible`?

Reply:
possibly additional test to ensure permissions restrict users that need to be restricted?